web/src/components/logs-table.css

```
@@ -1,47 +1,47 @@
+  padding-right: var(--pf-t--global--spacer--md);
   font-family: monospace;
   word-break: break-all;
   vertical-align: middle;
   width: 100%;
 }
 
 .lv-plugin__table .lv-plugin__table__resources {
-  --pf-v5-c-table--cell--FontSize: var(--pf-v5-global--FontSize--md);
-  margin-top: var(--pf-v5-global--spacer--xs);
+  --pf-v6-c-table--cell--FontSize: var(--pf-t--global--font--size--md);
+  margin-top: var(--pf-t--global--spacer--xs);
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)